integration_tests/support/commands.ts

```diff
@@ -13,3 +40,110 @@ Cypress.Commands.add('wiremockVerify', (requestPatternBuilder: RequestPatternBui
+  /* Update a Short Question Set Induction by answering the Do They Want To Work On Release question to YES to turn it
+   * into a Long Question Set Induction. Answer all the questions to get to the Check Your Answers page.
+   */
+  cy.task('stubGetInductionShortQuestionSet') // Long question set Induction with Hoping to work on release as YES
+  cy.visit(`/prisoners/${prisonNumber}/induction/hoping-to-work-on-release`)
+  // Hoping To Work On Release is the first page
+  Page.verifyOnPage(HopingToWorkOnReleasePage) //
+    .selectHopingWorkOnRelease(HopingToGetWorkValue.YES)
+    .submitPage()
+  // Qualifications List is the next page. Qualifications are asked on the short question set, so this will already have qualifications set
+  Page.verifyOnPage(QualificationsListPage) //
+    .submitPage()
+  // Additional Training is the next page. This is asked on the short question set, so this will already have answers set
+  Page.verifyOnPage(AdditionalTrainingPage) //
+    .submitPage()
+  // 'Has the prisoner worked before' is the next page. This is not asked on the short question set.
+  // Answer 'Yes' to create an Induction that has details of the prisoners previous work experience.
+  Page.verifyOnPage(WorkedBeforePage) //
+    .selectWorkedBefore(YesNoValue.YES)
+    .submitPage()
+  // Preview Work Experience types is the next page. This is not asked on the short question set.
+  Page.verifyOnPage(PreviousWorkExperienceTypesPage) //
+    .choosePreviousWorkExperience(TypeOfWorkExperienceValue.OFFICE)
+    .submitPage()
+  Page.verifyOnPage(PreviousWorkExperienceDetailPage) //
+    .setJobRole('Office junior')
+    .setJobDetails('Filing and photocopying')
+    .submitPage()
+  // Work Interests page is the next page. This is not asked on the short question set.
+  Page.verifyOnPage(FutureWorkInterestTypesPage) //
+    .chooseWorkInterestType(WorkInterestTypeValue.DRIVING)
+    .submitPage()
+  Page.verifyOnPage(FutureWorkInterestRolesPage) //
+    .setWorkInterestRole(WorkInterestTypeValue.DRIVING, 'Driving instructor')
+    .submitPage()
+  // Personal skills page is the next page. This is not asked on the short question set.
+  Page.verifyOnPage(SkillsPage) //
+    .chooseSkill(SkillsValue.TEAMWORK)
+    .submitPage()
+  // Personal Interests is the next page. This is not asked on the short question set.
+  Page.verifyOnPage(PersonalInterestsPage) //
+    .choosePersonalInterest(PersonalInterestsValue.SOCIAL)
+    .submitPage()
+  // Factors Affecting Ability To Work is the next page. This is not asked on the short question set.
+  Page.verifyOnPage(AffectAbilityToWorkPage) //
+    .chooseAffectAbilityToWork(AbilityToWorkValue.HEALTH_ISSUES)
+    .submitPage()
+  // Arrive on Check Your Answers page
+  Page.verifyOnPage(CheckYourAnswersPage)
+})
+
+const signInWithAuthority = (authority: 'EDIT' | 'VIEW') => {
+  cy.task('reset')
```

Comment:
These 11 lines are in pretty much every Cypress test we write, so I'm hoping over time we can refactor them to start using `cy.signInAsUserWithViewAuthorityToArriveOnPrisonerListPage()` or `cy.signInAsUserWithEditAuthorityToArriveOnPrisonerListPage()`

If nothing else, other than removing noise from the top of the tests, the cypress command name makes it much clearer as to what is happening and why we are setting up these stubs etc